package/MDAnalysis/units.py

```diff
@@ -343,16 +343,14 @@ def convert(x, u1, u2):
     try:
         ut1 = unit_types[u1]
     except KeyError:
-        raise ValueError(("unit '{0}' not recognized.\n" +
-                          len("ValueError: ")*[" "] +
-                          "It must be one of {1}.").format(u1, unit_types))
+        raise ValueError("unit '{0}' not recognized.\n"
+                          "It must be one of {1}.".format(u1, ", ".join(unit_types)))
```

Comment:
On 31 Mar, 2016, at 23:49, Jonathan Barnoud wrote:

> This is because we have the symbol for Angstrom as bytes and as unicode. This was done in purpose, but does anybody uses the utf-8 escape code for Å (b'\xe2\x84\xab')? Do we really need to support it

I don't think that we need to support it; we probably added it at some point to be complete but I honestly cannot remember.

> while we already support 'Angtrom', 'A', 'angstrom', and u'\u212b', and that we support b'\xe2\x84\xab' only for speed, force, and density units, but not length?

That's just inconsistent and not on purpose.